packages/sit-onyx/src/styles/mixins/input.scss

```diff
@@ -21,6 +21,27 @@
     // optional marker should be displayed at the very end of the label
     &.onyx-optional-marker {
       justify-content: space-between;
+
+      &::after {
+        display: none;
+      }
+    }
+  }
+
+  &__optional {
+    display: none;
+    font-family: var(--onyx-font-family);
+    font-weight: 400;
+    font-style: normal;
+    color: var(--onyx-color-text-icons-neutral-soft);
+    padding-left: var(--onyx-spacing-2xs);
+    hyphens: none;
+    margin-left: auto;
+  }
+
+  &.onyx-use-optional:not(:has(.onyx-required-marker)) {
```

Comment:
Also I think we broke the inheritance of the optional marker...when I change it to optional globally with the Storybook UI it does not work
![image](https://github.com/SchwarzIT/onyx/assets/67898185/7394a6fb-0770-494d-85a9-84f0d8831ef5)

Maybe we should use the `v-if="!props.required"` in the component instead

Reply:
But I think we are already using it

<img width="790" alt="image" src="https://github.com/SchwarzIT/onyx/assets/60856270/6a89cf13-9911-43fc-a57e-1e1f011c8ee7">